uaclient/config.py

```diff
@@ -117,6 +119,14 @@ def http_proxy(self):
     def https_proxy(self):
         return self.cfg.get("https_proxy")
 
+    @property
+    def apt_http_proxy(self):
```

Comment:
might as well use typing for everything we touch

What do you think about drive-by bring in the uaclient.defaults too for contract_url and security_url?
```diff
diff --git a/uaclient/config.py b/uaclient/config.py
index 80c1fc8b..acd753ed 100644
--- a/uaclient/config.py
+++ b/uaclient/config.py
@@ -10,7 +10,12 @@ import yaml
 from collections import namedtuple, OrderedDict
 
 from uaclient import status, util
-from uaclient.defaults import CONFIG_DEFAULTS, DEFAULT_CONFIG_FILE
+from uaclient.defaults import (
+    BASE_CONTRACT_URL,
+    BASE_SECURITY_URL,
+    CONFIG_DEFAULTS,
+    DEFAULT_CONFIG_FILE,
+)
 from uaclient import exceptions
 
 try:
@@ -104,27 +109,27 @@ class UAConfig:
         return []
 
     @property
-    def contract_url(self):
-        return self.cfg.get("contract_url", "https://contracts.canonical.com")
+    def contract_url(self) -> str:
+        return self.cfg.get("contract_url", BASE_CONTRACT_URL)
 
     @property
-    def security_url(self):
-        return self.cfg.get("security_url", "https://ubuntu.com/security")
+    def security_url(self) -> str:
+        return self.cfg.get("security_url", BASE_SECURITY_URL)
 
     @property
-    def http_proxy(self):
+    def http_proxy(self) -> "Optional[str]":
         return self.cfg.get("http_proxy")
 
     @property
-    def https_proxy(self):
+    def https_proxy(self) -> "Optional[str]":
         return self.cfg.get("https_proxy")
 
     @property
-    def apt_http_proxy(self):
+    def apt_http_proxy(self) -> "Optional[int]":
         return self.cfg.get("apt_http_proxy")
 
     @property
-    def apt_https_proxy(self):
+    def apt_https_proxy(self) -> "Optional[str]":
         return self.cfg.get("apt_https_proxy")
 
     def check_lock_info(self) -> "Tuple[int, str]":
```

Feel free to reject or tell me to put up a separate PR for this

Reply:
Good call! I can totally include that, maybe broken into a separate commit